webpack.config.js

```diff
@@ -22,17 +23,30 @@ export default {
       import: ['./javascripts/application.js', './stylesheets/application.scss']
     }
   },
+  experiments: {
+    outputModule: true
+  },
```

Comment:
This change is optional but configures webpack to output for `<script type="module">`

Using native modules allows `await import('/public/javascripts/another-module.mjs')` but we can drop this commit if you'd rather avoid it for now

It's been experimental since June 2021 as part of [`webpack@5.41.0`](https://github.com/webpack/webpack/releases/tag/v5.41.0)

Reply:
I'm fine with this 👍